docs/docs/shards/types.md

```diff
@@ -631,12 +631,14 @@ Its also known as map, data dictionary, or associative array. An example of a **
 
 ## Wire
 
-Type **Wire** is a sequence of shards (i.e., a sequence of [**ShardRef**](#shardref) or [**Shard**](#shardref) types).
+Type **Wire** represents a wire being passed as data. 
 
 !!! note
     Has keyword [`Wire`](https://docs.fragcolor.xyz/functions/values/#wire).
 
-In addition to the **Shard** type, [`(ForEach)`](https://docs.fragcolor.xyz/shards/General/ForEach/) also accepts a *sequence of **Shard** type values*, or in other words a `Wire` type, as data for its `:Apply` parameter.
+A **Wire** type thus consists of a sequence of shards (which make up the wire), their shared state (memory) context, name of the wire, and other properties that enable operations on the wire like scheduling it on a mesh, starting/stopping/pausing the wire, etc.
```

Comment:
It's kinda weird, we need a section explaining wire

Reply:
Am rewriting the shard and wire explanations in the Primer, will link to them from both the `ShardsRef` and `Wire` type sections.

Reply:
Added primer-links for `shard` and `wire` (primer content is WIP, but links are known).

Reply:
Added as part of the following commits
[#17be451956be18c7c73c006179039f9001ce4e72](https://github.com/fragcolor-xyz/shards/pull/351/commits/17be451956be18c7c73c006179039f9001ce4e72)
[#719f341aac5d3f6072bf7b2cfcb0cda633ba15e7](https://github.com/fragcolor-xyz/shards/pull/351/commits/719f341aac5d3f6072bf7b2cfcb0cda633ba15e7)

*2nd commit is a fixup, not able to rebase/fixup them together from local as rebase is pushing my local branch out of sync, hence have left the commits as they are.
@Kryptos-FR FYI

Reply:
Putting the local branch out of sync of remote is expected when rebasing. That's the point.
All you have to do is move the last commit between the 2 previous and use fixup during the interactive rebase. Then force-push.

Reply:
Thanks, that worked!
I think I was thrown off by the force-push dialog box message for some reason (even though I have done this earlier).